app/services/foreman_ansible/roles_importer.rb

```diff
@@ -1,24 +1,55 @@
 module ForemanAnsible
-  # Service to import roles from the filesystem.
-  # Should be extracted to a gem that both the proxy and Foreman can use.
+  # imports roles from smart proxy
```

Comment:
It is extra burden on the developer. It's so much easier for the user though, who doesn't have to care about proxy, proxy config files, or proxy plugins. 

I aim to alleviate the burden (duplication, really) between doing the same operation in proxy & Foreman via extracting this to a gem `ansible_roles_importer` or similar.